app/controllers/IndexController.scala

```diff
@@ -112,4 +121,7 @@ class IndexController extends BaseController {
     Ok(scala.xml.XML.loadString(new String(canonicalizer.canonicalizeSubtree(soapEnvelope))))
   }
 
+  def respondToSentry() = Action { implicit request =>
+    Ok("returnType=1\nid=" + request.body.asFormUrlEncoded.get("token"))
```

Comment:
Can we use `fakeMemberService.getResponseFor(member)` to get the attributes based on the cookie value? They should be simple to serialise

Reply:
I can look into it 👍